src/DotNet/Writer/UniqueChunkList.cs

```diff
@@ -57,13 +57,26 @@ public UniqueChunkList()
 		/// <inheritdoc/>
 		public override uint CalculateAlignment() {
 			uint alignment = base.CalculateAlignment();
-			chunks.Sort(DescendingComparer.Instance);
+
+			var keys = new KeyValuePair<int, Elem>[chunks.Count];
+			for (var i = 0; i < chunks.Count; i++)
+				keys[i] = new KeyValuePair<int, Elem>(i, chunks[i]);
+			Array.Sort(keys, DescendingStableComparer.Instance);
```

Comment:
Is `Array.Sort` a stable sort?
As I know it's not. Instead, `OrderBy` in LINQ is a stable sort

Reply:
The new code will perform a stable sort on `keys` as `DescendingStableComparer` first compares the alignment of the `Elem` in the `KeyValuePair` and then compares the original index if necessary.